Dockerfile

```diff
@@ -13,7 +13,6 @@ COPY main.go main.go
 COPY api/ api/
 COPY cache/ cache/
 COPY controllers/ controllers/
-COPY gitops/ gitops/
```

Comment:
I have a question. You're removing this directory and most files from it, but not all - suite_test.go remains there. Is that intentional?

Reply:
AIUI the entire directory should be deleted

Reply:
When I removed the dir in my test PR, the check finally worked:

https://github.com/redhat-appstudio/release-service/pull/390

https://github.com/redhat-appstudio/release-service/actions/runs/8100309735/job/22138001315?pr=390

Reply:
Very interesting. @happybhati can you remove the whole dir please? I don't think this line is the one causing the gosec error, but maybe having a suite_test file in a dir with nothing else in it is? :man_shrugging:

Reply:
@happybhati  I pushed a commit to remove the dir to your topic branch, I hope you don't mind.

And sure enough, it passes now: https://github.com/redhat-appstudio/release-service/actions/runs/8100401963/job/22138316767?pr=363

Reply:
Great finding, Thanks for removing it. @mmalina 🙏🏻